src/web3/ether.ts

```diff
@@ -7,7 +7,7 @@ const erc20Abi = [
 const provider = new ethers.providers.JsonRpcProvider("https://rpc.fuse.io");
 
 const gooddollar = new ethers.Contract(
-  "0x495d133B938596C9984d462F007B676bDc57eCEC",
+  "0x79BeecC4b165Ccf547662cB4f7C0e83b3796E5b3",
```

Comment:
this shouldnt be hard coded

Reply:
I can include this in config file will that work?

Reply:
we probably need to rewrite the architecture ie
```
const sdk = new PaymentSDK(optional settings, g$ contract address, network)
sdk.generatePaymentLink
sdk....
```
@TomasGonzalez

Reply:
> we probably need to rewrite the architecture ie
> 
> ```
> const sdk = new PaymentSDK(optional settings, g$ contract address, network)
> sdk.generatePaymentLink
> sdk....
> ```
> 
> @TomasGonzalez

@sirpy I see what you mean, but is it really necessary to receive the g$ contract address and network as a parameter? I think this can fetch it to the API using the @gooddollar/protocol npm package and that way it won't make sense to make it work as a class...

Reply:
yes it would be best to get it from @gooddollar/goodprotocol
just remember that there are multiple environments (fuse,staging,production) and multiple networks(mainnet and fuse) each with different contract address.
i guess we can hard code it to use by default `network="production"` at the moment. but keep `network` as input variable and use it to read from `deployment.json`